README.md

```diff
@@ -39,9 +41,9 @@ puppet module install puppetlabs-dsc
 
 ## Usage
 
-### Refresh Mode Must be Disabled
+### LCM RefreshMode Must be Disabled
```

Comment:
So I just verified locally - no need to touch the LCM anymore!  Submitting a PR to remove the PowerShell checks / update tests.